postcss.config.cjs

```diff
@@ -0,0 +1,6 @@
+const tailwindcss = require('tailwindcss');
+const autoprefixer = require('autoprefixer');
```

Comment:
do we really need these variables?  like you are assigning values in the new variable which should be done by only one line. and u are consuming memories.

Remember the big-O notation.???

Reply:
Hahaha! Actually, I wanted to use ES imports which add a variable too but it seems PostCSS doesn't allow it but I found that using require in the code block is not recommended especially by Eslint check [Here](https://eslint.org/docs/rules/global-require) 
That's why I added my require on the top